org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/util/concurrent/DeferredOperationQueue.java

```diff
@@ -55,10 +57,18 @@ public class DeferredOperationQueue<E> {
   private volatile IFuture<?> m_flushJobFuture;
 
   public DeferredOperationQueue(String transactionMemberId, int batchSize, long maxDelayMillis, Consumer<List<E>> batchOperation) {
-    this(transactionMemberId, batchSize, maxDelayMillis, batchOperation, null);
+    this(transactionMemberId, batchSize, maxDelayMillis, batchOperation, null, null);
   }
 
   public DeferredOperationQueue(String transactionMemberId, int batchSize, long maxDelayMillis, Consumer<List<E>> batchOperation, Supplier<RunContext> runContextSupplier) {
+    this(transactionMemberId, batchSize, maxDelayMillis, batchOperation, runContextSupplier, transactionMemberId);
+  }
+
+  public DeferredOperationQueue(String transactionMemberId, int batchSize, long maxDelayMillis, Consumer<List<E>> batchOperation, String flushJobName) {
```

Comment:
Do we need this constructor? `null` can be passed if the runContextSupplier is not relevant.